### PR TITLE
added getLatestAnalysis

### PR DESCRIPTION
**Pull request by nathanBrenner**

### CODE REVIEW

File: lib/reports/reports.js [hunk 1]
```diff
@@ -103,6 +119,7 @@ export class Reports {
 }
 
 const getAnalysisEndpoint = '/v1/report/getAnalysis'
+const getLatestAnalysisEndpoint = '/v1/report/getLatestAnalysis'
```
Comment (dan9186) [inline lib/reports/reports.js]: I don't believe this is the right endpoint. Let me see if I can find it.

Comment (nathanBrenner) [reply lib/reports/reports.js]: v1/animal/getLatestAnalysis ?

Comment (dan9186) [reply lib/reports/reports.js]: Ok. The endpoint is `v1/scanner/getLatestAnalysisStatus`. Given that, we will want to move this method to a new resource object called `Scanner`.

A corresponding version in Ionic can be found here https://github.com/ion-channel/ionic/blob/master/scanner.go#L80. The shape of the response is available here https://github.com/ion-channel/ionic/blob/master/scanner/analysis_status.go#L36.

Comment (nathanBrenner) [reply lib/reports/reports.js]: great thanks

**Pull request merged**